### PR TITLE
added jvm memory data into stats operation response

### PR DESCRIPTION
**Pull request by bmaschio**

### CODE REVIEW

File: include/runtime.iol [hunk 1]
```diff
@@ -108,6 +108,12 @@ type Stats:void {
 		.systemLoadAverage:double //< System load average
 		.version:string //< OS version
 	}
+
+	.jvm:void{
```
Comment (fmontesi) [inline include/runtime.iol:112]: `jvm` is not very intuitive for me. What about `memory`? That way we don't need the `Mem` suffix in the subnodes.

```
memory {
  free:long
  used:long
  total:long
}
```

File: packages/runtime.ol [hunk 2]
```diff
@@ -108,6 +108,11 @@ type Stats:void {
 		.systemLoadAverage:double //< System load average
 		.version:string //< OS version
 	}
+	.jvm:void{
```
Comment (fmontesi) [inline packages/runtime.ol:111]: See first comment.

File: javaServices/coreJavaServices/src/main/java/joliex/lang/RuntimeService.java [hunk 3]
```diff
@@ -325,4 +326,11 @@ private void stats_files( Value stats ) {
 			stats.setFirstChild( "maxCount", unixBean.getMaxFileDescriptorCount() );
 		}
 	}
+
+	private void stats_jvm( Value stats ) {
+		Runtime runtime = Runtime.getRuntime();
```
Comment (fmontesi) [inline javaServices/coreJavaServices/src/main/java/joliex/lang/RuntimeService.java:331]: See first comment.

**Review by fmontesi — CHANGES_REQUESTED** [review]

File: include/runtime.iol (hunk 1 above)
Comment (balintPixis) [reply include/runtime.iol:112]: the idea was to be able to get more JVM related info in a future but I can change it not a problem

**Review by fmontesi — APPROVED** [review]

**Pull request merged**